patches/integration.diff

```diff
@@ -104,18 +104,22 @@ Index: code-server/lib/vscode/src/vs/workbench/browser/parts/dialogs/dialogHandl
 ===================================================================
 --- code-server.orig/lib/vscode/src/vs/workbench/browser/parts/dialogs/dialogHandler.ts
 +++ code-server/lib/vscode/src/vs/workbench/browser/parts/dialogs/dialogHandler.ts
-@@ -144,11 +144,12 @@ export class BrowserDialogHandler implem
+@@ -143,12 +143,15 @@ export class BrowserDialogHandler implem
+ 
  	async about(): Promise<void> {
  		const detailString = (useAgo: boolean): string => {
- 			return localize('aboutDetail',
+-			return localize('aboutDetail',
 -				"Version: {0}\nCommit: {1}\nDate: {2}\nBrowser: {3}",
-+				"code-server: v{4}\nCode: {0}\nCommit: {1}\nDate: {2}\nBrowser: {3}",
++			return localize('aboutCodeServerDetail',
```

Comment:
Yup.  Really we could just leave out the `localize` call here since no one will be localizing it and it does not have anything that needs localizing anyway but...just in case I guess.

Reply:
My thinking exactly!

Reply:
![image](https://user-images.githubusercontent.com/3806031/161156521-6dec96ec-6a52-4f13-8486-21ff6b242ff0.png)